frontend/amundsen_application/static/js/components/Badges/BadgeBrowseList/styles.scss

```diff
@@ -0,0 +1,44 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+@import 'typography';
+
+.badges-browse-section {
+  margin: 32px 0;
```

Comment:
Let's try to make this values a multiple of the grid or use the spacer variables